manifests/limits.pp

```diff
@@ -0,0 +1,25 @@
+# == Class: lightblue::limits
+#
+# Configures limits in /etc/security/limits.conf
+#
+# === Parameters
+#
+# [*config_dir*]
+#   The director in which jcliff configuration files are deployed.
+#
+# === Variables
+#
+# Module requires no global variables.
+#
+class lightblue::limits (
+    $soft_nproc = undef,
+    $hard_nproc = undef
```

Comment:
Depends on how the class is used.  For our internal use, yes we'll set it in hiera.  Note the lightblue::limits class must be referenced somewhere, probably in a profile class.